recipes/TextToSpeech.py

```
@@ -417,6 +429,65 @@ def run(self, state: dict):
+                sarvam_request.update(
+                    {
+                        key: value
+                        for key, value in {
+                            "speaker": speaker,
+                            "pace": state.get("sarvam_tts_pace"),
+                            "temperature": state.get("sarvam_tts_temperature"),
+                        }.items()
+                        if value is not None
+                    }
+                )
+                response = requests.post(
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)